up42/order.py

```diff
@@ -18,7 +18,9 @@ def _translate_construct_parameters(order_parameters):
     order_parameters_v2 = deepcopy(order_parameters)
     params = order_parameters_v2["params"]
     data_product_id = order_parameters_v2["dataProduct"]
-    if not params.get("displayName", None):
```

Comment:
This can be simplified to
```python
default_name = f"{data_product_id} order"
order_parameters_v2["displayName"] = params.get("displayName", default_name)
```